Editor/CompileCesiumForUnityNative.cs

```diff
@@ -284,6 +321,8 @@ private static string GetDirectoryNameForPlatform(BuildTargetGroup platformGroup
         {
             if (IsEditor(platformGroup, platform))
                 return "Editor";
+            else if (IsIOS(platformGroup, platform))
+                return "iOS";
```

Comment:
Is this needed? If I'm not mistaken, `platformGroup.ToString()` will return `"iOS"` as well.

Reply:
Joseph pointed out offline that it actually returns `iPhone` because there are multiple enums with the same value. Resolving.